docs/UserGuide.md

```diff
@@ -234,28 +263,112 @@ Oops! There seems to be an error, please check your fields again.
 
 Deletes the data entry at the index-th position of the currently displayed list.
 
-Format: `delete INDEX...`
+Format: `delete INDEX [INDEX...]`
+
+Parameter: `INDEX [INDEX...]`
+* Index of a fosterer is displayed in the list that is  obtained by the `find/list` command.
+* At least one index must be provided.
+* Index must be a positive integer: 1, 2, 3, …
+
+<div markdown="span" class="alert alert-primary">
+  :bulb: <b>Tip:</b>
+Multiple indices are allowed for mass deletion, each index separated by a white space.
+</div>
+
+* Duplicate indices are ignored.
+* Extraneous white spaces are also ignored.
 
-Parameter: `INDEX...`
-* Index of a fosterer is shown in the list obtained by the `find/list` command.
-* Must be a positive integer 1, 2, 3, …
-* Multiple indexes are allowed for mass deletion,  each index separated by a white space.
-* `delete` without an index is valid if and only if there exists only one data entry in the current list.
 
 Examples:
 * `list` followed by `delete 2` deletes the 2nd fosterer in the address book
-* `find Jerry` followed by `delete 1` deletes the 1st fosterer in the results of the find command
+* `find Jerry` followed by `delete 1` deletes the 1st fosterer in the result list of the find command
 * `list` followed by `delete 1 3 7` deletes the 1st, 3rd and 7th fosterers in the address book
+* `list` followed by `delete 3 3 3 3` deletes the 3rd fosterer in the address book
+
 
 Expected output (success):
+`list` followed by `delete 1 3 7`
 ```agsl
-Fosterers Jerry Chee, John Doe, and Mary Ann are successfully deleted!
+3 fosterers deleted:
```

Comment:
Right now Im getting something like this: 
Deleted Fosterers:
Bernice Yu; Phone: 99272758; Email: berniceyu@example.com; Address: Blk 30 Lorong 3 Serangoon Gardens, #07-18; Housing: Landed; Availability: NotAvailable; Animal name: Bobby; Animal type: current.Dog; Tags: [reliable][experienced], 
Irfan Ibrahim; Phone: 92492021; Email: irfan@example.com; Address: Blk 47 Tampines Street 20, #17-35; Housing: HDB; Availability: nil; Animal name: nil; Animal type: nil; Tags: [experienced]



but if you are planning to change the output, its fine!

Reply:
yea HAHA that was the original output prior to this PR! but i felt including the number of fosterers deleted will be helpful, so i added it in